src/buckets/components/context/csvUploaderProvider.tsx

```diff
@@ -152,11 +153,20 @@ export const CsvUploaderProvider: FC<Props> = React.memo(({children}) => {
               )
             }
             time = table.getColumn('_time')?.[i] ?? Date.now()
-            value = table.getColumn('_value')?.[i] ?? field ?? ''
+            table.columnKeys
+              // Matches _value, _value ('number'), _value ('string'), etc. to find value
+              // https://github.com/influxdata/giraffe/blob/master/giraffe/src/utils/fromFlux.ts#L62
+              .filter(c => /_value( \('\w*'\))?/g.test(c))
+              .forEach(c => {
+                if (table.getColumn(c)?.[i]) {
+                  value = table.getColumn(c)[i]
+                  valueColumn = c
+                }
+              })
```

Comment:
I think we might be able to simplify this by simply passing in the datatype as the second parameter in the `table.getColumn` function:

https://github.com/influxdata/giraffe/blob/master/giraffe/src/utils/newTable.ts#L25

Reply:
Turns out I was running and giraffe is 💩  with this.  Looks good to me!